.stylelintrc

```diff
@@ -2,6 +2,9 @@
     "extends": ["stylelint-config-standard"],
     "customSyntax": "postcss-less",
     "rules": {
+        "string-quotes": null,
+        "import-notation": null,
```

Comment:
the value has to be `"string"`, not `null`
https://stylelint.io/user-guide/rules/import-notation/